config/routes.rb

```diff
@@ -2,266 +2,268 @@
       end
     end
-  end
 
-  namespace :api do
-    namespace :v1 do
-      resources :planning_applications, only: %i[index create show] do
+    namespace :public do
+      resources :planning_applications, only: [] do
         member do
-          get :decision_notice
+          get "decision_notice"
         end
-        resources :validation_requests, only: :index
-        resources :description_change_validation_requests, only: %i[index update show]
-        resources :replacement_document_validation_requests, only: %i[index update show]
-        resources :additional_document_validation_requests, only: %i[index update show]
-        resources :documents, only: %i[show]
-        resources :other_change_validation_requests, only: %i[index update show]
-        resources :fee_change_validation_requests, only: %i[index update show]
-        resources :ownership_certificate_validation_requests, only: %i[index update show]
-        resources :ownership_certificates, only: %i[create]
-        resources :red_line_boundary_change_validation_requests, only: %i[index update show]
-        resources :pre_commencement_condition_validation_requests, only: %i[index update show]
-        resources :neighbour_responses, only: :create
       end
+    end
 
-      resources :local_authorities, only: %i[show], param: :subdomain
+    namespace :api do
+      namespace :v1 do
+        resources :planning_applications, only: %i[index create show] do
+          member do
+            get :decision_notice
+          end
+          resources :validation_requests, only: :index
+          resources :description_change_validation_requests, only: %i[index update show]
+          resources :replacement_document_validation_requests, only: %i[index update show]
+          resources :additional_document_validation_requests, only: %i[index update show]
+          resources :documents, only: %i[show]
+          resources :other_change_validation_requests, only: %i[index update show]
+          resources :fee_change_validation_requests, only: %i[index update show]
+          resources :ownership_certificate_validation_requests, only: %i[index update show]
+          resources :ownership_certificates, only: %i[create]
+          resources :red_line_boundary_change_validation_requests, only: %i[index update show]
+          resources :pre_commencement_condition_validation_requests, only: %i[index update show]
+          resources :neighbour_responses, only: :create
+        end
+
+        resources :local_authorities, only: %i[show], param: :subdomain
 
-      resources :documents, only: :show do
-        get :tags, on: :collection
+        resources :documents, only: :show do
+          get :tags, on: :collection
+        end
       end
     end
+
+    get :healthcheck, to: proc { [200, {}, %w[OK]] }
   end
 
-  get :healthcheck, to: proc { [200, {}, %w[OK]] }
+  constraints Constraints::ConfigSubdomain do
+    mount BopsConfig::Engine => "/"
+  end
 end
```

Comment:
could we maybe extract the main BOPS routes to a separate file as is getting too long?